inc/classes/class_xml.php

```diff
@@ -149,7 +149,7 @@ public function get_tag_content($tag_to_grab, $input, $replace_html = true) {
      * @param int       $level
      * @return string
      */
-    public function level2tab($level) {
+    private function level2tab($level) {
```

Comment:
Referenced in XMLtest.php L46

Reply:
Deleted in this PR also: https://github.com/lansuite/lansuite/pull/190/files#diff-5509b988ae8d72113b9cace28e758387L44